src/components/Picklist/helpers/__test__/isOptionVisible.spec.js

```diff
@@ -10,31 +9,45 @@ describe('isOptionVisible', () => {
             top: container.top,
             bottom: container.bottom,
         }));
+    });
 
-        elements = [
+    it("should return true when element is within container's visible bounds", () => {
```

Comment:
don't push commits without run prepush because it exists for a reason
in this case this is a lint error and you commit it, then fix it using single quotes

Reply:
Normally i run yarn lint and yarn test before do the push cos throug 3g connection is impossible to let prepush do the job